content/en/docs/reference/glossary/Quantity.md

```diff
@@ -1,18 +1,21 @@
 ---
-title: Gibibyte
-id: GiB
-date: 2023-01-21
-full_link: https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/
+title: Quantity
+id: Qty
+date: 2023-01-24
+full_link: https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/
 short_description: >
-  GiB stands for Gibibyte. 1 GiB is defined as 1024³ bytes
+ Quantity contains GB and GiB unit. 
+GiB: Gibibyte 
```

Comment:
I have added these extra blocks for more clarity. if we need to follow the strict yaml pattern as used in other file i can change it like that. Also the reference link that is added is by the approval of @sftim in the issue section. Thank you.